contents/blog/launch-week-universe-of-new-features.md

```diff
@@ -0,0 +1,78 @@
+date: 2022-03-14
+title: PostHog Launch Week I: A Universe of New Features
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ..
+featuredImageType: full
+author: ["andy-vandervell"]
+categories: ["Product updates"]
+---
+
+PostHog made huge strides in 2021. To name just a few landmarks, we: 
+
+- Raised a [$15 million Series B ahead of schedule](/blog/why-we-raised-a-15m-series-b-ahead-of-schedule)
+- Became one of [YC's top-valued companies](/blog/yc-top-companies)
+- Rebranded our website [in just four weeks](/blog/postmortem-rebrand)
+- Migrated from [postgresql to ClickHouse](/blog/how-we-turned-clickhouse-into-our-eventmansion)
+- Achieved product market fit for PostHog Scale
+
+All of this (and more) lead to huge growth. Users grew by 4x, the number of open source contributors doubled, we acquired numerous reference customers, and went from zero revenue to significant revenue.
```

Comment:
```suggestion
All of this (and more) lead to huge growth. Users grew by 4x, the number of open source contributors doubled, we acquired [numerous reference customers](/customers), and went from zero revenue to significant revenue.
```

I'd prob cut the revenue mention. We generally don't talk about financial stuff publicly and this makes it clear that it was first year, thus overshadowing other achievements.

Reply:
The revenue reference I took from one of James' recent ones, so figured it was safe. Can take out if people feel strongly. As we're not putting a number on it, it does invite speculation somewhat...